bootstrap/src/lib/cache/redis.ts

```diff
@@ -14,23 +14,27 @@ const DEFAULT_CACHE_REDIS_TIMEOUT = 500 // Timeout in ms
 const DEFAULT_CACHE_MAX_AGE = 1000 * 30 // Maximum age in ms
 
 const env = process.env
-const defaultOptions = () => ({
+
+export type RedisOptions = ClientOpts & { maxAge: number; timeout: number }
+
+export const defaultOptions = (): RedisOptions => ({
   host: env.CACHE_REDIS_HOST || DEFAULT_CACHE_REDIS_HOST,
-  port: env.CACHE_REDIS_PORT || DEFAULT_CACHE_REDIS_PORT,
-  path: env.CACHE_REDIS_PATH || DEFAULT_CACHE_REDIS_PATH,
-  url: env.CACHE_REDIS_URL || DEFAULT_CACHE_REDIS_URL,
+  port: Number(env.CACHE_REDIS_PORT) || DEFAULT_CACHE_REDIS_PORT,
+  path: env.CACHE_REDIS_PATH || ((DEFAULT_CACHE_REDIS_PATH as unknown) as undefined),
+  url: env.CACHE_REDIS_URL || ((DEFAULT_CACHE_REDIS_URL as unknown) as undefined),
```

Comment:
This is a bandaid over the fact that we are setting these defaults to `null` rather than what the @types/Redis expects: `undefined`. 

We could assert it as undefined, but it gives this warning:
```
Conversion of type 'null' to type 'undefined' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.ts(2352)
```

Reply:
Ok, let's set it as undefined then 🤞. Sounds like a non-issue but that lib spanked me few times unexpectedly.